### DISCUSSION TITLE
While configuring dbeaver eclipse development environment, I ran into a difficulty at step 13

### DISCUSSION THREAD

**Discussion by Keith-Xing** [dbeaver/dbeaver#34346] [Q&A]
I am following the official tutorial, the link is https://dbeaver.com/docs/dbeaver/Develop-in-Eclipse/. When proceeding to step 13, my interface is showing a different example than the tutorial, does anyone know how to fix this please? Thank you very much!
My interface is like this, there is only a green checkmark, and I can't do anything about it, clicking on finish in this interface doesn't respond, I can only click on cancel.
![企业微信截图_17182667631263](https://github.com/dbeaver/dbeaver/assets/129652250/da0c8450-8504-44f8-b2a4-45fc50c534fb)
but the tutorial looks like this:
![import-maven-3](https://github.com/dbeaver/dbeaver/assets/129652250/3de6ffed-dc61-4981-83ac-331e4d227ea9)

**Comment by E1izabeth** [comment]
What Eclipse version do you use? I would suggest trying RCP 2024-06

**Reply by Keith-Xing** [reply]
it is RCP 2023-06

**Reply by E1izabeth** [reply]
Better using latest version.

**Comment by TB-HunBoka** [comment]
Did you install the basic Eclips called "Eclips IDE for Java Developers"? I did the same and I remember having the same problem. I had no experience with eclips and it was not well documented what RCP means.

You need to start the Eclips installer and scroll down until you see Eclipse IDE for RCP and RAP Developers.
![kép](https://github.com/dbeaver/dbeaver/assets/126617861/aafad6a0-caf8-4a7f-a3a0-58bbf8440bb0)

**Reply by Keith-Xing** [reply]
I install the rcp and rap

**Reply by TB-HunBoka** [reply]
Then sorry, I was trying to install it for couple of days, and I clearly remember having the same window, but could not remember how did I solve it. Let me know if you still have the issue with the above-mentioned latest version and then I will try to reproduce what I did.

**Comment by TB-HunBoka** [comment]
I was playing with installing different versions of eclipse. With the latest version of eclipse, I didn't get this window at all and was able to install and configure it (mostly). Could you install it too? Then you can mark E1izabeth's comment as best answere.
